src/backend/distributed/shared_library_init.c

```diff
@@ -519,7 +519,7 @@ RegisterCitusConfigVariables(void)
 					 "be higher. This setting does not apply if the input format is "
 					 "binary."),
 		&SkipJsonbValidationInCopy,
-		true,
+		false,
```

Comment:
I'd change such that the value is not visible users via `GUC_NO_SHOW_ALL `. What do you think?

I think we don't have anything in the citus docs related to `citus. skip_jsonb_validation_in_copy`. Could you double check? I guess we don't want to document it before we fix the issue.

Reply:
This makes sense. I updated the PR. I also could not find a reference in the docs as expected.